source/clear-linux/tutorials/multi-boot/multi-boot-restore-bl.rst

```diff
@@ -14,43 +15,48 @@ longer the default boot loader. To restore it, follow the steps below.
 
    .. figure:: figures/multi-boot-restore-bl-1.png
 
-      Figure 1: |CL|: Console
+      Figure 1: |CL|: Console.
 
 #. Log in as *root*.
 
+   .. code-block:: bash
+
+      sudo -s
```

Comment:
We don't need lines 22-24.  Since the user logged in as "root", they don't need to "sudo -s".